module-2/backend/week_two.md

```diff
@@ -25,30 +29,52 @@ end
 ```
 
 Now how would you find the owner of the team with an id of 4?
+Team.where(owner_id: 4)
 
 3. What do they allow you to do?
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+teacher has_many :students
+student belongs_to :teacher
+
 8. Define foreign key, primary key, and schema.
+Primary key is the direct ID of that object
+Foreign key is the key in a relationship table that links to the other object. example being team having "owner_id"
+schema is were all the tables were setup tells you what is in each table "column" and what type of it is. string integer etc etc
 9. Describe the relationship between a foreign key on one table and a primary key on another table.
+team having "owner_id" in its values. whereas the Team has its own unique ID too
```

Comment:
This doesn't really describe to me what a primary key and foreign  key are and how they relate.